pkg/client/services.go

```diff
@@ -84,7 +85,11 @@ func (c *services) Create(svc *api.Service) (result *api.Service, err error) {
 	if needNamespace && len(namespace) == 0 {
 		namespace = api.NamespaceDefault
 	}
-	err = c.r.Post().Namespace(namespace).Resource("services").Body(svc).Do().Into(result)
+	// TODO(a-robinson): Remove this explicit timeout and the ones below once
+	// #5180 is in.
+	request := c.r.Post()
+	request.Timeout(4 * time.Minute)
```

Comment:
I guess? There's nothing special about this number and it should be getting ripped out later this week, but I'll change it if that's what's blocking the merge.